api-specs/test/package.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
You have to add the `api-specs` folder to the list of workspaces.

https://github.com/commercetools/commercetools-docs-kit/blob/5336b9977734fdda6f4a1f3359cfd52ea4a68c06/package.json#L7